src/lib/navigation/navigation-mode-input.interface.ts

```diff
@@ -0,0 +1,7 @@
+import {WizardComponent} from '../components/wizard.component';
+import {NavigationMode} from './navigation-mode.interface';
+
+/**
+ * Type of [[WizardComponent]]'s navigationMode input.

```

Comment:
Ideally the comment should also explain what the different values mean, i.e. what is an allows string?

Reply:
Added improved comments in 589880a